### ISSUE TITLE
About Chinese and description errors

### ISSUE THREAD

**Issue by johnbitcn** [zanfab/docsify-plugin-flexible-alerts#4]
I want use chinese label ,I rewrote here
```javascript
var l = (t || "").match(new RegExp(e + ":(([\\w\\s\\u4e00-\\u9fa5]*))"));
```

```javascript
/<\s*blockquote[^>]*>(?:<p>|[\S\n]*)?\[!(\w*)((?:\|[\w*:[\u4e00-\u9fa5\w\s]+)*?)\]([\s\S]*?)(?:<\/p>)?<\s*\/\s*blockquote>/g,
```
and in README.md  Have an error

```
iconVisibility:false
```

**Comment by zanfab** [comment]
Thanks @johnbitcn for your hint concerning Chinese labels.
Can you please provide an example of the issue with and without your fix?

I'm not sure what you mean with the error in README.md.
Can you provide further details?

**Comment by johnbitcn** [comment]
I read in README.md file, at here"iconVisibility:felse" ,but it did't work. when I read code file, it is "iconVisibility:hidden" .
![001](https://user-images.githubusercontent.com/1518771/53313212-55d8f100-38f3-11e9-8346-b0ff348d7506.png)

then when I use chinese label, it can't be show. so I fix it. because  in regular expression, "\w" is only stands "a-z" and "A-Z"

```markdown
> [!TIP|label:自定义标题]
>  可以显示中文标题。
```
**before:**
![003](https://user-images.githubusercontent.com/1518771/53314748-f252c180-38fa-11e9-9ffa-6419c687b7de.png)

**after fix:**
![002](https://user-images.githubusercontent.com/1518771/53314772-0dbdcc80-38fb-11e9-860d-174bc354d087.png)

I hope express is clearly.
my english like shit, so don't mind

**Comment by zanfab** [comment]
Fixed in release 1.0.2
